### ISSUE TITLE
Feature Request: Run containers from powershell script

### ISSUE THREAD

**Issue by HugoLS** [mozilla/multi-account-containers#2209]
Hi,

I need to start multiple sessions of firefox in parallel for testing purpose with the help of powershell.

For now, i'm able to start multiple instances of firefox with a specific profile. 
But i would prefere to be able to use containers instead. Launch only one instance of firefox, and let containers handle the sessions.

Is there any way to use containers from PS script?

Thank you

**Comment by Nothing4You** [comment]
this extension allows you to do this by using custom protocol handlers: https://addons.mozilla.org/en-US/firefox/addon/open-url-in-container/

**Comment by dannycolin** [comment]
Duplicate of #365

**Comment by unhammer** [comment]
I think this would be solved by https://bugzilla.mozilla.org/show_bug.cgi?id=1774127
